CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.0.1] - November 21, 2022
+- Added support of manual vertical `TooltipPosition`.
```

Comment:
Instead of highlighting as parameter name just write it directly like this -> tooltip position. Or if want to use parameter name then make proper sentence for parameter.